index.html

```diff
@@ -1586,6 +1586,11 @@ <h3>Key types and formats</h3>
 A public key can be used as a <a>verification method</a>.
         </p>
 
+<p>
+A <a>verification method</a> MUST contain a single verification material property
+(for example <code>publicKeyJwk</code> or <code>publicKeyBase58</code>, not both).
```

Comment:
```suggestion
A <a>verification method</a> MUST NOT contain multiple verification material properties. 
For example, expressing key material in a <a>verification method</a> using both 
<code>publicKeyJwk</code> and <code>publicKeyBase58</code> at the same time 
is prohibited.
```

Might want to specify this as a prohibition instead.

Reply:
yes, i had it that way, but changed it... was not sure of the conventions for this.

Reply:
I prefer what I suggested above. Let's wait to see if others have better suggestions than the two options put forward.

Reply:
Oh wait, you already accepted it... bleh -- Github now doesn't show that the suggested change was accepted by you. :(

In any case, I'm happy with it as is... let's see if the new text confuses in a different way. :)

Reply:
I had what you suggested before, I committed... I think it's clearer. I was just not sure what the standard practice for "allow-list" vs "deny-list" in specs is.... in security "allow-list" is generally better, but in this case, I agree its less readable.

Reply:
Oh, I see what you're saying now.

From a spec text perspective, as a general rule, whatever is more clear and resistant to being misinterpreted is preferred. Consistency on how you say things throughout the spec is also a consideration, but less of one than the previous sentence.

In general, allow-lists are preferred over deny-lists... except when you really don't want the thing on the deny-list to happen because it will lead to catastrophic failures. That's more of an exception than a rule... and fundamentally, what goes in is whatever you get the group to come to consensus on... and unfortunately that's not always consistent throughout the spec. 

So, Editors are tasked with keeping all the language as consistent as possible throughout the spec while keeping at least all the things above in mind.